### PR TITLE
feat(timtetable state indicators): timtetable state indicators, open,…

### PR DESCRIPTION
**Pull request by RaySage**
… closed or no indicator

### CODE REVIEW

**Review by mauriciokin — DISMISSED** [review]
legit

**Review by Philip-Philippou1 — DISMISSED** [review]

File: packages/forms-web-app/src/controllers/projects/examination-timetable.js [hunk 1]
```diff
@@ -131,7 +150,7 @@ const getExaminationTimetable = async (req, res) => {
 
 	let nextDeadline = null;
 	const nextDeadlineEvent = events.find((event) => {
-		return !event.closed && event.typeOfEvent === 'Deadline';
+		return event.eventState.value === 'open' && event.typeOfEvent === 'Deadline';
```
[the remark this answers is not in the document]
Comment (RaySage) [reply packages/forms-web-app/src/controllers/projects/examination-timetable.js:153]: As we agreed we are leaving this as it is

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**